src/Eto/Forms/Controls/DocumentControl.cs

```diff
@@ -3,7 +3,7 @@
 /// <summary>
 /// Arguments for the <see cref="DocumentControl"/> to get the current page.
 /// </summary>
-public class DocumentPageEventArgs : EventArgs
+public class DocumentPageEventArgs : CancelEventArgs
```

Comment:
Not all events using `DocumentPageEventArgs` support canceling, however when we derive all of them from `CancelEventArgs` it might give a false indication that they are. 

How about creating a separate class just for `DocumentPageClosingEventArgs`?

Reply:
Applied your comment, you can have a look in this [commit](https://github.com/picoe/Eto/pull/2574/commits/1f4f36902896e7def2622c8ef7c0220ad62a57eb)